package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "windows-iana",
-  "version": "0.0.0-development",
-  "description": "A small tool to convert Windows time zones to IANA",
+  "version": "0.0.1-development",
```

```suggestion
  "version": "0.0.0-development",
```

No need to change this. The project uses [semantic-release](https://github.com/semantic-release/semantic-release).